### PR TITLE
chore(webpack): adds #TESTS alias for ./tests

### PR DESCRIPTION
**Pull request by juliangieseke**
NT

### CODE REVIEW

**Comment by DanielMSchmidt** [comment]
a) Why do we need this?
b) Is there a place in the current code base where we would use this?

**Comment by juliangieseke** [comment]
Alfred test this

File: webpack/webpack.config.babel.js [hunk 1]
```diff
@@ -163,7 +163,8 @@ module.exports = {
       PluginTestUtils: absPath("src/js/plugin-bridge/PluginTestUtils"),
       "#EXTERNAL_PLUGINS": externalPluginsDir,
       "#PLUGINS": absPath("plugins"),
-      "#SRC": absPath("src")
+      "#SRC": absPath("src"),
+      "#TESTS": absPath("tests")
```
Comment (bstavroulakis) [inline webpack/webpack.config.babel.js]: @juliangieseke Where will this be used? Just curious, you can merge.

Comment (juliangieseke) [reply webpack/webpack.config.babel.js]: just check where this PR is referenced :)

**Comment by juliangieseke** [comment]
Alfred test this

**Comment by nLight** [comment]
Alfred test this

**Comment by DanielMSchmidt** [comment]
Alfred test this please :)

**Comment by DanielMSchmidt** [comment]
Alfred test this please :)

**Pull request merged**